input.go

```diff
@@ -92,6 +92,29 @@ func IsMouseButtonPressed(mouseButton MouseButton) bool {
 	return uiDriver().Input().IsMouseButtonPressed(driver.MouseButton(mouseButton))
 }
 
+// GamepadGUID returns a string with the uuid.
+//
+// GamepadGUID is concurrent-safe.
+//
+// GamepadGUID always returns an empty string on browsers and mobiles.
```

Comment:
Comment that `GamepadGUID is concurrent-safe.`

Reply:
It does have that comment?